core/resources/build.properties

```diff
@@ -1,6 +1,6 @@
 
 # The OpenRocket build version
-build.version=20.11.alpha.16
+build.version=22.xx.BETA
```

Comment:
Might as well commit to 22.0 as the release number. The .xx is too ambiguous, so hammer it home :)

Reply:
The plan is to replace the xx with the month number when this release is final.

Reply:
Also: don't use capitals for beta, but make it e.g. 22.02.beta.01, so it's in sync with how the software update checker works